### ISSUE TITLE
Author/series name mismatch

### ISSUE THREAD

**Issue by wizzy99** [DobyTang/LazyLibrarian#1657]
Current Version: 37efad16ddce03a37677bd0192fea5718461abf4

This may be new behavior, but I'm seeing a few Author and Series mismatch messages in the logs when updating authors. Some of them are big differences, others are simply spaces (E.J. vs E. J.).

What's the suggested cleanup process when these are encountered?

lazylibrarian.log:06-Nov-2018 10:07:12 - WARNING :: AUTHORUPDATE : importer.py:addAuthorToDB:181 : Authorname mismatch for 7771479 [E. J. Koh][E.J. Koh]
lazylibrarian.log:06-Nov-2018 11:41:18 - WARNING :: AUTHORUPDATE : importer.py:addAuthorToDB:181 : Authorname mismatch for 298475 [Alexander Romanovich Belyaev][Alexei Yurchak]
lazylibrarian.log.2:04-Nov-2018 01:36:04 - WARNING :: AUTHORUPDATE : bookwork.py:getWorkSeries:880 : Name mismatch for series 216898, [Justice League (2011) (Single Issues)][The New 52 Justice League]

**Comment by wizzy99** [comment]
Got it.  So that error will pop every time the author is refreshed, right, unless it gets changed back to whatever it was originally?

**Comment by philborman** [comment]
Yes, it's just a warning. Hopefully as goodreads info improves over time the problems should reduce. Could suppress the message on a refresh fairly easily, only warn on first run?

**Comment by wizzy99** [comment]
Doesn't really matter, I was mostly curious as to whether it was a new check.  I know now that there isn't really anything to do about it so feel free to close this up.
